canvas_sdk/utils.py

```diff
@@ -23,6 +23,18 @@ def validate_attr_is_acceptable(value, acceptable_values=[], allow_none=True):
                 raise AttributeError("%s must be one of %s" % (v, acceptable_values))
 
 
+def validate_any(param_choices, *args, **kwargs):
+    """
+    If all of the arguments are falsy (e.g. None or blank strings), an
+    Attribute error is raised, otherwise nothing is returned. param_choices
+    should be list of variable names for feedback on a validation failure.
+    """
+    if not any(args):
```

Comment:
Comment states if all arguments are false it will raise the error, which appears to be the validation standard for the SDK. Ergo if any argument is true, it will not raise an error.